README.md

```diff
@@ -22,64 +22,23 @@ should make sure you choose strong passwords for `AUTH_CREDENTIALS`.
 
 ## Installation
 
-To run as an app on Aptible:
+To deploy Kibana as an App on Enclave:
 
- 1. Create an app in your [Aptible dashboard](https://dashboard.aptible.com) for Kibana. In the
-    steps that follow, we'll use &lt;YOUR_KIBANA_APP_HANDLE&gt; anywhere that you should substitute the
-    actual app handle the results from this step in the instructions.
+   aptible apps:create "$HANDLE" 
+   aptible deploy --app "$HANDLE" --docker-image aptible/kibana AUTH_CREDENTIALS=username:password DATABASE_URL=http://user:password@example.com
```

Comment:
There are many usability problems here:

- This is completely unreadable once rendered to Markdown. You need to add code fences (i.e. triple backquotes) above and below these two lines to ensure your newlines are interpreted as newlines (and to get code formatting). Consider using the rich view of the diff in Github once you've made changes in order to confirm this looks like what you expect (if you don't know how to do that, let me know, and I'll show you).
- Can we explain what `$HANDLE` is, and describe what these commands do? 
- Can we explain what `AUTH_CREDENTIALS` do?
- Can we wrap `AUTH_CREDENTIALS=...` in quotes for users that might (intentionally or not) use a shell-reserved keyword in their credentials?
- Can we wrap `DATABASE_URL` in quotes (largely for consistency, but the same reasons apply)
- Can we explain what `DATABASE_URL` is, and use a  `DATABASE_URL` that remotely resembles the one an Aptible user is probably going to use? This repo is mainly designed to be used by Enclave users: we should simply tell them that `DATABASE_URL` is their connection URL rather than expect them to figure that out by indicating that their database should be "reachable there".